### PR TITLE
added hi

### PR DESCRIPTION
**Pull request by oeuclidesneto**
Put comment hi on homepage

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 <h1>Pages#home</h1>
 <p>Find me in app/views/pages/home.html.erb</p>
+<p>hi</p>
```
Comment (Vietvu95) [inline app/views/pages/home.html.erb:3]: good stuff

**Review by Vietvu95 — APPROVED** [review]
do a better job

**Review by nchhomtythya — COMMENTED** [review]
good

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (oeuclidesneto) [reply app/views/pages/home.html.erb:3]: all good

**Review by nchhomtythya — COMMENTED** [review]
hi

**Pull request merged**